modules/serverless-rn-1-7-0.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * serverless/release-notes.adoc
+
+[id="serverless-rn-1-7-0_{context}"]
+
+= Release Notes for Red Hat {ServerlessProductName} 1.7.0
+
+[id="new-features-1-7-0_{context}"]
+== New features
+* {ServerlessProductName} 1.7.0 is now Generally Available (GA) on {product-title} 4.3 and newer versions. In previous versions, {ServerlessProductName} was a Technology Preview.
+* {ServerlessProductName} now uses Knative Serving 0.13.2.
+* {ServerlessProductName} now uses Knative `kn` CLI 0.13.2.
+* {ServerlessProductName} now uses Knative Serving Operator 0.13.2.
+* Knative Eventing is now available as a Technology Preview with {ServerlessProductName}. {ServerlessProductName} uses Knative Eventing 0.13.2.
+* Active/passive high availability (HA) is now available by default on {ServerlessProductName}.
```

Comment:
@mgencur I guess we should call out the individual components here instead that are HA for serving?

Reply:
TODO: call out `autoscaler-hpa`, `controller`, Kourier Gateway, Kourier Controller and Knative Activator components that are HA for Knative Serving